src/Messages.js

```diff
@@ -2,6 +2,11 @@
 import { defineMessages } from 'react-intl';
 
 export default defineMessages({
+    documentTitle: {
+        id: 'documentTitle',
+        description: 'The title of the page as it appears in the browser tab',
+        defaultMessage: 'Advisor {subapp} | Red Hat Insights'
```

Comment:
This should actually be '{subapp} - Advisor | Red Hat Insights'

The `subapp` should also be capitalized. ex) `Topics - Advisor | Red Hat Insights`